app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -301,6 +317,12 @@ class ThreadListAdapter(
         const val SWIPE_ANIMATION_THRESHOLD = 0.15f
         private val CARD_ELEVATION = 6.toPx().toFloat()
         private val CARD_CORNER_RADIUS = 12.toPx().toFloat()
+        private val THREAD_MARGIN_LARGE = 12.toPx()
+        private val THREAD_MARGIN_OTHER = 8.toPx()
+        private val DATE_TOP_MARGIN_LARGE = 4.toPx()
+        private val DATE_BOTTOM_MARGIN_LARGE = 4.toPx()
+        private val DATE_TOP_MARGIN_OTHER = 8.toPx()
```

Comment:
Why don't you get them directly from the dimens?

Reply:
We need access to a context to do so, and passing a context up to here is kinda unpractical. What do you think? Should we still do it?

Reply:
Why would you need to pass a context ?, you have views available, so you can use their contexts